server/src/classes/client.js

```diff
@@ -64,6 +70,7 @@ class Scanner {
     this.scanning = true;
   }
   cancelScan() {
+    // TODO: Should we set this.scanRequest to '' ?
```

Comment:
I want to keep the scan request intact so the crew and staff can know what the cancelled request was.

```suggestion 
```